Source/Scene/Scene.js

```diff
@@ -427,6 +434,34 @@ define([
         this.debugShowFramesPerSecond = false;
 
         /**
+         * This property is for debugging only; it is not for production use.
+         * <p>
+         * Displays depth information for the indicated view frustum slice.
+         * </p>
+         *
+         * @type Boolean
+         *
+         * @default false
+         */
+        this.debugShowGlobeDepth = false;
+
+        /**
+         * This property is for debugging only; it is not for production use.
+         * <p>
+         * Indicates which frustum slice will have depth information displayed.
+         * </p>
+         *
+         * @type Number
+         *
+         * @default 1
+         */
+        this.debugShowGlobeDepthFrustum = 1;
+
+        this._debugGlobeDepthTextures = [];
```

Comment:
Moved functionality to GlobeDepth pending further optimization in 697bd05294c5a848588a095afc7b60645ad6b353